Sources/RequestBuilder.swift

```diff
@@ -113,12 +119,19 @@ struct MultipleResourceRequest<R: GarageRequest, D: Decodable>: ResourceRequest
     let configuration: GarageConfiguration
 
     func response(from object: Any, urlResponse: HTTPURLResponse) throws -> Response {
-        guard let resource: [D] = try? decodeArray(object) else {
-            throw ResponseError.unexpectedObject(object)
-        }
 
         let parameters = headerParameters(from: urlResponse)
-        return GarageResponse(resource: resource, totalCount: parameters.totalCount, linkHeader: parameters.linkHeader)
+        if let rootKeyPath = baseRequest.rootKeyPath {
+            guard let resource: [D] = try! decodeArray(object, rootKeyPath: KeyPath(rootKeyPath)) as [D]? else {
```

Comment:
Fix it.
https://github.com/slightair/GarageClientSwift/pull/3/files#diff-1d696d720be823d13aae2986bcc884f5R125